instrumentation/pulsar/pulsar-2.8/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/pulsar/v2_8/PulsarClientTest.java

```diff
@@ -671,4 +672,30 @@ void testConsumePartitionedTopicUsingBatchReceive() throws Exception {
                               });
                         }));
   }
+
+  @Test
+  void testSendMessageWithTxn() throws Exception {
+    String topic = "persistent://public/default/testSendMessageWithTxn";
+    admin.topics().createNonPartitionedTopic(topic);
+    producer =
+        client
+            .newProducer(Schema.STRING)
+            .topic(topic)
+            .sendTimeout(0, TimeUnit.SECONDS)
+            .enableBatching(false)
+            .create();
+    Transaction txn =
```

Comment:
Could you name the variable `transaction` instead of `txt`. We usually don't use abbreviations for local variables. I think this is suggested by google java style guide.